pandas/core/strings.py

```diff
@@ -2371,8 +2371,14 @@ def cat(self, others=None, sep=None, na_rep=None, join="left"):
         join : {'left', 'right', 'outer', 'inner'}, default 'left'
             Determines the join-style between the calling Series/Index and any
             Series/Index/DataFrame in `others` (objects without an index need
-            to match the length of the calling Series/Index). To disable
-            alignment, use `.values` on any Series/Index/DataFrame in `others`.
+            to match the length of the calling Series/Index).
+
+            **Note**: Since version 1.0.0, index alignment is performed when
+            `others` is a Series/Index/DataFrame (or a list-like containing
+            one). To disable alignment (the behavior in and before v0.23.0),
+            use `.values` in `others`, or, alternatively, convert `others` to a
```

Comment:
We discourage public use of `.values` so should remove that reference (`to_numpy()` is the canonical way)

Reply:
Understood, I've removed the `.values` references.